woocommerce/woocommerce
# Fix code typo in Orders Analytics

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

I noticed that there is a code typo in the Orders Analytics code. The code is about `$coupons`, but it is written `$mapped_data[ $product['order_id'] ]['coupons']`. I believe logically it should be `$mapped_data[ $coupon['order_id'] ]['coupons']`.

I believe the typo comes from copying and pasting from another line in the same file.

In this PR, I fix the typo. 

Note that there should be no behavior change. This is because in that same code processing, `$product['order_id']` and  `$coupon['order_id']` would give the same order ID.

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check out trunk branch, and view Orders Analytics. 
2. Check out this PR branch, and view Orders Analytics.
3. Both Orders Analytics should be the same.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
